teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './person'
+class Teacher
+  def initialize(specialization, age, name = 'unknown')
+    super(age, name)
+    @specialization = specialization
+  end
```

Comment:
```suggestion
   def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

- [ ] Kindly initialize and add `parent_permission` in the arguments in super, your code should look like the suggestion above 👍🏿

Reply:
hi @prolajumokeoni 
Thanks a lot for your review 
I removed parent_permission because this is the teacher's class not the student and anyways the teacher will always be above 18 and the can-use method is also overridden so it will be redundant code